ClosedXML_Tests/Excel/Worksheets/XLWorksheetTests.cs

```diff
@@ -748,6 +748,98 @@ void AssertPicturesAreEqual(IXLWorksheet ws1, IXLWorksheet ws2)
+                    Assert.AreEqual(original.ErrorValueReplacement, copy.ErrorValueReplacement);
+                    Assert.AreEqual(original.EmptyCellReplacement, copy.EmptyCellReplacement);
+                    Assert.AreEqual(original.AutofitColumns, copy.AutofitColumns);
+                    Assert.AreEqual(original.PreserveCellFormatting, copy.PreserveCellFormatting);
+                    Assert.AreEqual(original.ShowGrandTotalsColumns, copy.ShowGrandTotalsColumns);
+                    Assert.AreEqual(original.ShowGrandTotalsRows, copy.ShowGrandTotalsRows);
+                    Assert.AreEqual(original.FilteredItemsInSubtotals, copy.FilteredItemsInSubtotals);
+                    Assert.AreEqual(original.AllowMultipleFilters, copy.AllowMultipleFilters);
+                    Assert.AreEqual(original.UseCustomListsForSorting, copy.UseCustomListsForSorting);
+                    Assert.AreEqual(original.ShowExpandCollapseButtons, copy.ShowExpandCollapseButtons);
+                    Assert.AreEqual(original.ShowContextualTooltips, copy.ShowContextualTooltips);
+                    Assert.AreEqual(original.ShowPropertiesInTooltips, copy.ShowPropertiesInTooltips);
+                    Assert.AreEqual(original.DisplayCaptionsAndDropdowns, copy.DisplayCaptionsAndDropdowns);
+                    Assert.AreEqual(original.ClassicPivotTableLayout, copy.ClassicPivotTableLayout);
+                    Assert.AreEqual(original.ShowValuesRow, copy.ShowValuesRow);
+                    Assert.AreEqual(original.ShowEmptyItemsOnColumns, copy.ShowEmptyItemsOnColumns);
+                    Assert.AreEqual(original.ShowEmptyItemsOnRows, copy.ShowEmptyItemsOnRows);
+                    Assert.AreEqual(original.DisplayItemLabels, copy.DisplayItemLabels);
+                    Assert.AreEqual(original.SortFieldsAtoZ, copy.SortFieldsAtoZ);
+                    Assert.AreEqual(original.PrintExpandCollapsedButtons, copy.PrintExpandCollapsedButtons);
+                    Assert.AreEqual(original.RepeatRowLabels, copy.RepeatRowLabels);
+                    Assert.AreEqual(original.PrintTitles, copy.PrintTitles);
+                    Assert.AreEqual(original.SaveSourceData, copy.SaveSourceData);
+                    Assert.AreEqual(original.EnableShowDetails, copy.EnableShowDetails);
+                    Assert.AreEqual(original.RefreshDataOnOpen, copy.RefreshDataOnOpen);
+                    Assert.AreEqual(original.ItemsToRetainPerField, copy.ItemsToRetainPerField);
+                    Assert.AreEqual(original.EnableCellEditing, copy.EnableCellEditing);
+                    Assert.AreEqual(original.ShowRowHeaders, copy.ShowRowHeaders);
+                    Assert.AreEqual(original.ShowColumnHeaders, copy.ShowColumnHeaders);
+                    Assert.AreEqual(original.ShowRowStripes, copy.ShowRowStripes);
+                    Assert.AreEqual(original.ShowColumnStripes, copy.ShowColumnStripes);
```

Comment:
Can't we reuse the same method here and in `XLPivotTableTests`?

Reply:
Or maybe it's time to add the FluentAssertions package and leverage the `.ShouldBeEquivalentTo` method

Reply:
I created a new `PivotTableComparer` and consolidated this.